pkg/manager/manager_test.go

```diff
@@ -47,6 +47,119 @@ var _ = Describe("manger.Manager", func() {
 		close(stop)
 	})
 
+	Describe("PreStart", func() {
+		It("should call the function on each Component", func(done Done) {
+			// Init the manager
```

Comment:
you should use `By("blah")` here if you find yourself leaving comments like this -- it has similar behavior when reading the code, and if something goes wrong when running the test, you get nicer output.